esi/src/lib.rs

```diff
@@ -243,7 +272,7 @@ impl Processor {
         )
     }
 
-    /// Process an ESI document from a [`crate::Reader`], handling includes and directives
+    /// Process an ESI document from a [`esi::Reader`], handling includes and directives
```

Comment:
Resolves doc generation warning

```suggestion
    /// Process an ESI document from a [`Reader`], handling includes and directives
```